mq/src/buildant/rules.xml

```diff
@@ -275,7 +275,8 @@
 		 version="true"
 		 use="true"
 		 windowtitle="${JMQ_SOFTWARE_NAME_SHORT}, 5.1 API Specification"
-		 doctitle="${JMQ_SOFTWARE_NAME_SHORT}, 5.1 API Specification">
+		 doctitle="${JMQ_SOFTWARE_NAME_SHORT}, 5.1 API Specification"
+         additionalparam="--add-modules java.xml.ws">
```

Comment:
per [README](https://github.com/eclipse-ee4j/openmq/blob/3de016ceca7a74c649b5d6be53a7e96d6a84e7f4/mq/README#L7) MQ is built with JDK 1.8, but there is no such option `--add-modules` yet in 8.

Also in https://github.com/eclipse-ee4j/openmq/issues/421#issuecomment-424403189 it was said

> Although the official build runs against Java SE 8 now

Is this change required for *hk2.config-generator update to 5.1.0-RC2*? Otherwise the commit brings more changes than declared in message I think.

Reply:
> Ant build fix

What is broken here that needs fixing?
I'd prefer to have commit messages corresponding to changes. Then when inspected without github's PR metadata it's also clear what (from diff) was done with reasoning for the modification in comment (within code repository alone). IMO *Ant build fix* is no better than *fix*, as one has to check everything anyway.